src/d3a/models/market/two_sided.py

```diff
@@ -52,20 +52,30 @@ def __init__(self, time_slot=None, bc=None, notification_listener=None, readonly
         super().__init__(time_slot, bc, notification_listener, readonly, grid_fee_type,
                          grid_fees, name, in_sim_duration=in_sim_duration)
 
+    @property
+    def _class_name(self):
+        return "TwoSidedPayAsBid"
```

Comment:
```suggestion
        return "TwoSidedMarket"
```

Not guaranteed to always be pay-as-bid at this point.